### PR TITLE
PLF-7244 : Upgrade to Shindig 2.5.2 and remove fork

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
* rename package name org.gatein.shindig to org.apache.shindig
* updates to use the new Shindig API
* analytics feature has been moved to features-extra : https://github.com/apache/shindig/commit/40813e87297851de02b2656cbd0cc426d5dca3c7
---> it requires to install a new jar (shindig-extras-2.5.2.jar) to use this feature. By default I think we can simply not use it, it will be up to the customer to add it if he needs it.

### CODE REVIEW

File: gadgets/core/src/main/java/org/exoplatform/portal/gadget/core/ExoDefaultSecurityTokenGenerator.java [hunk 1]
```diff
@@ -44,18 +50,17 @@ public ExoDefaultSecurityTokenGenerator() throws Exception {
...
+            values.put(BlobCrypterSecurityToken.Keys.OWNER.getKey(), owner);
+            values.put(BlobCrypterSecurityToken.Keys.VIEWER.getKey(), viewer);
+            values.put(BlobCrypterSecurityToken.Keys.TRUSTED_JSON.getKey(), "");
+            BlobCrypterSecurityToken t = new BlobCrypterSecurityToken(container, null, null, values);
 
-            return t.encrypt();
-        } catch (IOException e) {
-            log.error(e.getMessage(), e);
-        } catch (BlobCrypterException e) {
+            return t.getContainer() + ':' + blobCrypter.wrap(t.toMap());
```
Comment (aboughzela) [inline gadgets/core/src/main/java/org/exoplatform/portal/gadget/core/ExoDefaultSecurityTokenGenerator.java]: Any reason to add container to encrypts and signs a blob ?

File: gadgets/core/src/main/javascript/features/online-features.txt [hunk 2]
```diff
@@ -1,6 +1,3 @@
 #
 #  THe list of features that require running in ONLINE mode
 #
-
-features/analytics/feature.xml
-features/com.google.gadgets.analytics/feature.xml
```
Comment (aboughzela) [inline gadgets/core/src/main/javascript/features/online-features.txt]: should  add doc issue for this change (remove analytics feature  )  and how customer can add it if he needs it.

File: gadgets/core/src/main/java/org/exoplatform/portal/gadget/core/ExoDefaultSecurityTokenGenerator.java (hunk 1 above)
Comment (thomasdelhomenie) [inline gadgets/core/src/main/java/org/exoplatform/portal/gadget/core/ExoDefaultSecurityTokenGenerator.java]: That's how the token is expected to be formatted : https://github.com/apache/shindig/blob/1.0.x/java/common/src/main/java/org/apache/shindig/auth/BlobCrypterSecurityTokenDecoder.java#L101-L105

File: gadgets/core/src/main/javascript/features/online-features.txt (hunk 2 above)
Comment (thomasdelhomenie) [inline gadgets/core/src/main/javascript/features/online-features.txt]: Right, I just wanted others opinions on this point. I will add it in the issue.

File: gadgets/core/src/main/java/org/exoplatform/portal/gadget/core/ExoDefaultSecurityTokenGenerator.java (hunk 1 above)
Comment (aboughzela) [inline gadgets/core/src/main/java/org/exoplatform/portal/gadget/core/ExoDefaultSecurityTokenGenerator.java]: thx, it's clear now

File: gadgets/core/src/main/javascript/features/online-features.txt (hunk 2 above)
Comment (aboughzela) [inline gadgets/core/src/main/javascript/features/online-features.txt]: ok, should be treated on other issue

**Comment by aboughzela** [comment]
LGTM

**Pull request merged**